index.js

```diff
@@ -13,6 +13,8 @@ function GzipFilter(inputTree, options) {
 
   this.keepUncompressed = options.keepUncompressed;
   this.appendSuffix = options.hasOwnProperty('appendSuffix') ? options.appendSuffix : true;
+  this.inputEncoding = options.inputEncoding || null;
```

Comment:
Worth including a comment on why this is necessary -- it's not obvious just from looking at the code what's going on.